woocommerce/woocommerce
# Remove "Feeling Stuck" tooltip

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR removes the prompt that used to be displayed to collect feedback after users interacted with the legacy product creation page for a few minutes without saving their work.

![feeling-stuck](https://github.com/woocommerce/woocommerce/assets/3380168/ad49209e-c3c2-4ab4-aa2b-3bb9937f4fce)

Closes #40130.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install and activate WC-beta-tester, go to Tools > WCA Test Helper, search the option `woocommerce_ces_product_feedback_shown`, and delete it if it exists.
2. Go to the legacy product editor. Wait for 7 minutes. The prompt shouldn't be displayed.
3. If you want to see the prompt, you should checkout `trunk`, be sure that you deleted the option `woocommerce_ces_product_feedback_shown` (step 1), go to the legacy product editor, add a product name without saving, and wait for 7 minutes. If you don't want to wait for 7 minutes, you can use the branch `dev/40130_remove_feeling_stuck_tooltip_test` that I prepared for testing, which sets a 5-second wait.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
